internal/pkg/agent/application/info/agent_id.go

```diff
@@ -189,10 +189,12 @@ func loadAgentInfo(forceUpdate bool, logLevel string, createAgentID bool) (*pers
 	if err := idLock.TryLock(); err != nil {
 		return nil, err
 	}
-	defer idLock.Unlock()
```

Comment:
add `//nolint:errcheck // comment` 😛

Reply:
yep, same issue as you stumbled upon with your PR earlier, will update